client/app/components/pages/GeneHome.vue

```diff
@@ -1152,7 +1166,6 @@ export default {
             }
           }
         }
-        this.sendAnalysisToClin();
```

Comment:
This method was called while analyzing every gene. It resulted in calling several other functions in the process and there were many network requests made: `promiseExportAnalysisVariant()` -->  `promiseExportFlaggedVariant()` --> `promiseExportVariants()`  --> `_promiseCreateExportRecord()`  -->  `promiseGetVariantExtraAnnotations()`  -->  `_promiseFormatRecord()`

The advantage that this line of code provided was: Each variant was annotated. So when a user reviewed a variant (eg. marked it significant), it did not have to make a network request, and the findings page in clin instantly displayed the reviewed variant.  

After deleting this line, the extra annotate variant request is made only after the variant is reviewed and only for that respective variant. So only one request is made thereby not overloading the backend. This causes a slight delay while the response is returned, hence the `report-findings` page does not immediately display the reviewed variant. 

As a solution for this, I have added a progress dialog. It notifies the user that the variant interpretation is in progress. 

<img width="1034" alt="Screen Shot 2020-07-14 at 5 49 20 PM" src="https://user-images.githubusercontent.com/16284713/87490382-68dc8600-c5fa-11ea-9a02-8493d5fa89da.png">